docs/build-repro.md

```diff
@@ -167,3 +167,50 @@ An example `policy.json`:
 ```
 
 Any source type is supported, but how to pin a source depends on the type.
+
+## `SOURCE_DATE_EPOCH`
+[`SOURCE_DATE_EPOCH`](https://reproducible-builds.org/docs/source-date-epoch/) is the convention for pinning timestamps to a specific value.
+
+The Dockerfile frontend supports consuming the `SOURCE_DATE_EPOCH` value as a special build arg, since Dockerfile 1.5 (BuildKit 0.11).
+
+```console
+buildctl build --frontend dockerfile.v0 --opt build-arg:SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct) ...
+```
+
+The `buildctl` CLI does not automatically propagate the `$SOURCE_DATE_EPOCH` environment value from the client host to the `SOURCE_DATE_EPOCH` build arg.
+However, higher level build tools, such as Docker Buildx (>= 0.10), may automatically capture the environment value.
+
+The build arg value is used for:
+- the `created` timestamp in the [OCI Image Config](https://github.com/opencontainers/image-spec/blob/main/config.md#properties)
+- the `created` timestamp in the `history` objects in the [OCI Image Config](https://github.com/opencontainers/image-spec/blob/main/config.md#properties)
+- the `org.opencontainers.image.created` annotation in the [OCI Image Index](https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys)
```

Comment:
also mention other exporters than image?

Reply:
mentioned `local` and `tar`